src/components/dialog/dialog.pw.tsx

```diff
@@ -496,6 +497,19 @@ test.describe(
       await checkAccessibility(page);
     });
 
+    // TODO: (FE-6245) - this test should be failing color contrast accessibility test with grey background
+    test.skip("DialogWithStepSequence mock component should pass accessibility checks", async ({
```

Comment:
**comment** (non-blocking) - I don't think the test description is very accurate here. I would have thought the important point is that the `hasGreyBackground` prop passes accessibility tests, with the "step sequence" part not being very relevant here? Yet the title only means the step sequence and not the `hasGreyBackground`.

Note that with the TODO comment saying this should actually fail, I'm not sure what the actual intention is, since obviously we don't want failing accessibility tests!

Reply:
I may have not explained myself very well in the comment sorry - this test was added to address the issue raised regarding the color contrast issues with the green text in `StepSequence` and the previous default grey background. When making sure that the test failed when `hasGreyBackground` prop was passed, the test still passed. I added the ticket number to look into possible issues with the `checkAccessibility()` util, this is also why the test is being skipped at the moment.

Reply:
thanks @nuria1110 - I now see this is my own fault for misunderstanding the original ticket 😅 I'd thought we already had white as the unchangeable default and you were adding a grey option, but now I look more closely I see that we're changing the default to white, and adding a prop to get back to the grey colour. Which of course will change the background color of most dialogs used by consumers, but that's fine if it's what the DS team are asking for 👍 

Regarding the test, I'd make it so that it's testing without the grey background, and ensuring the accessibility is OK. That should presumably pass so imo it's better to not skip it. We should still keep your comment though so we know to investigate it not failing even when the grey background is used and the contrast should fail.